### ISSUE TITLE
misspelling of subscriptions (susbcriptions) in error message

### ISSUE THREAD

**Issue by plasma-vision** [microsoft/vscode-tools-for-ai#1584]
Note the misspelling of subscriptions (susbcriptions) in the error message below: 

Does this occur consistently? 
Repro steps:

1.
2.

Action: azureAccount.onSessionsChanged
Error type: 123
Error Message: Unknown Error retrieving susbcriptions from Azure Account extension

Version: 0.10.0
OS: win32
OS Release: 10.0.19044
Product: Visual Studio Code
Product Version: 1.67.2
Language: en

<details>
<summary>Call Stack</summary>

```
_.<anonymous> extension.js:2:2324475
s extension.js:2:2321343
```

</details>

**Comment by sevillal** [comment]
@tbombach this seems to be a duplicate of the onSessionsChanged issue, could you please take a look?

**Comment by shsuman** [comment]
Fixed in 0.18.0
